docs/UserGuide.md

```diff
@@ -560,6 +595,17 @@ Searches for a public address.
 
 Format: `searchpa pa/PUBLIC_ADDRESS`
 
+
+<box type="tip" seamless>
+
+The field PUBLIC_ADDRESS is not cap sensitive.
```

Comment:
Standardise to case sensitive? Or doesn't matter

Reply:
apologies, I shall use case-insensitive